frontend/src/components/Button/index.tsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import styles from "./style.module.css";
+
+interface ButtonProps {
+  variant: "primary" | "secondary";
```

Comment:
Thoughts on making the variant optional & default to "primary"?

Reply:
Clean. Will do